conf/scylla.yaml

```diff
@@ -622,3 +622,14 @@ strict_is_not_null_in_views: true
 # minimum_replication_factor_warn_threshold:  3
 # maximum_replication_factor_warn_threshold: -1
 # maximum_replication_factor_fail_threshold: -1
+
+# Guardrails to warn about or disallow creating a keyspace with specific replication strategy.
+# Each of these 2 settings is a list storing replication strategies considered harmful.
+# The replication strategies to choose from are:
+# - simple,
+# - local,
+# - network_topology,
+# - everywhere_topology
```

Comment:
I want to raise a question (I don't know the answer, maybe you or @tzach gave this some thought):
Do we want to invent new names like "network_topology", or could/should we just use the original names of the strategies like NetworkTopologyStrategy, verbatim, to make it more clear what exactly is being restricted?

Reply:
These names are not entirely made up, they just come from the source code:
https://github.com/scylladb/scylladb/pull/15399/files#diff-9be2281a24993f93765810ad5298ac0e57ba9a58601b4025a09727f9fec331cdR1226
tho I agree the strings are arbitrary and it's better to have them more user-friendly. That being said, which string you prefer - "Simple" or "SimpleStrategy"? Strategy is kinda redundant, no?

Reply:
What do you mean it "comes from the source code" - it comes from the **new** source code, where you made up those names :-)

I agree that SimpleStrategy is a long and ugly name, but in CQL you already need to write that name, so why should it be different in scylla.yaml? Isn't it simpler that once someone sees in the documentation a cool new NewReplicationStrategy they can use this name - and not variants of it they have to guess - everywhere that needs a replication strategy name?

Reply:
Right, now I am using full names.